mosip_master/data_upgrade/1.2.0.1_to_1.3.0/data-uploader.py

```
@@ -45,6 +45,7 @@ def getCurrentDateTime():
 
 
 def get_seed_value():
+  seed_value = None
   conn = psycopg2.connect(database="mosip_master", user = args.dbusername, password = args.dbpassword, host = args.dbhost, port = args.dbport)
   cursor = conn.cursor()
   cursor.execute("select id from master."+args.table+" order by id desc limit 20")
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)